### ISSUE TITLE
P#65163 Redirection fails with errors

### ISSUE THREAD

**Issue by jmaas-onoffice** [onOfficeGmbH/oo-wp-plugin#226]
# Current state
I've set up https://2.wptest.onofficeweb.com/ with version [3.0-46-g9f542ba4](https://github.com/onOfficeGmbH/oo-wp-plugin/pull/211#issuecomment-1110947154). The detail page has _no_ parent page.

The detail page is linked correctly, e.g. https://2.wptest.onofficeweb.com/detail/35-og119/. When I break the URL, e.g. https://2.wptest.onofficeweb.com/detail/35-og/, I get a blank page with only the following error messages:

```
Warning: Cannot modify header information - headers already sent by (output started at /www/htdocs/w01cc1b0/2.wptest.onofficeweb.com/htdocs/wp-content/themes/neve/header.php:10) in /www/htdocs/w01cc1b0/2.wptest.onofficeweb.com/htdocs/wp-includes/pluggable.php on line 1355

Warning: Cannot modify header information - headers already sent by (output started at /www/htdocs/w01cc1b0/2.wptest.onofficeweb.com/htdocs/wp-content/themes/neve/header.php:10) in /www/htdocs/w01cc1b0/2.wptest.onofficeweb.com/htdocs/wp-includes/pluggable.php on line 1358
```

This also happens when the title is not in the URL. Then https://2.wptest.onofficeweb.com/detail/35/ works correctly, but when I go to https://2.wptest.onofficeweb.com/detail/35-og119/ I get the same error message as above, when I should have been redirected to https://2.wptest.onofficeweb.com/detail/35/.

# Desired state
The plugin should actually redirect to the correct URL.

**Comment by LongTrong-exe** [comment]
I tested it on my wordpress, it still works, I can't reproduce it. I will ask our tester to check it.
But I think you should check your wordpress environment.

https://user-images.githubusercontent.com/101325858/165565898-0f9c206b-7641-48bc-9d6d-dfedec63b580.mp4

**Comment by jmaas-onoffice** [comment]
Could you try it on https://2.wptest.onofficeweb.com/? You should have access to that test instance.

**Comment by jmaas-onoffice** [comment]
Fixed as of [3.0-59-g9b3be5ce](https://github.com/onOfficeGmbH/oo-wp-plugin/pull/211#issuecomment-1115812696).
